docs/md-vs-ipynb.md

```diff
@@ -0,0 +1,40 @@
+
+## Is the experience of reading and writing any different between `.md` and `.ipynb`?
+
+In general, there isn't any difference between reading and writing these two formats.
+If you use [Jupytext](https://jupytext.readthedocs.io/en/latest/) or [JupyterLab-MyST](https://github.com/jupyter-book/jupyterlab-myst), then your experience in a Jupyter interface will be the exact same. It is only when you _save the notebook_ that it gets written to `.md` or `.ipynb`. Here is the process in either case:
+
+One exception to this is **loading a notebook from disk**. If using a `.md` file, you may need to take an extra UI action to load the `.md` file as a notebook (instead of as a text file). In addition, if you're using a `.md` file then the outputs will be empty every time you load it (because they aren't stored with the file).
+
+Finally, if you're using a `.md` file, then you'll be able to edit it from any text editor, rather than requiring a notebook UI to load. You can read and understand the structure of a `.md` file on your own without needing to render it in a browser.
+
+## When to lean towards Markdown (`.md`)
+
+- **Version control clarity:** When **tracking changes** to the notebook's content (code and narrative) in a collaborative setting is crucial. Markdown's readability makes `diff`ing straightforward.
+- **Human-focused authoring:** If you prioritize writing and reviewing the **source content** in a highly readable, human-editable format.
+- **Collaborative review:** For workflows involving frequent **reviews of the code and explanations**, where output changes are less critical to the review process.
+- **Integration with text-based workflows:** When you intend to use notebooks with tools that excel with Markdown, such as static site generators or documentation pipelines.
+- **When re-executing your content is important or easy:** When you want to re-execute your content at build-time, markdown-based formats are usually easier to work with. This allows you to ensure your code is reproducible, so long as it isn't extremely complex or costly to do execute. This is often the preferred case in tutorials for libraries where notebooks are regularly executed in CI to ensure continued functionality.
+
+## When to lean towards `.ipynb`
+
+- **Output preservation:** When you need to store and share the **results of code execution** (text, tables, figures) without requiring re-running.
+- **Out-of-the-box Jupyter UI support:** When you cannot assume that your reader can install `jupytext` or `jupyterlab-myst`.
+- **Computational intensity:** For notebooks with **lengthy or resource-intensive computations**, saving outputs via `.ipynb` avoids redundant runs.
```

Comment:
Making a note for now -- this is sort of true. If you're using `jupyter-book`, then the MD file also caches the outputs. But, this is less robust to e.g. adding a new code cell above the existing outputs (well, it's correct to invalidate the cache, but it means the outputs are lost).